src/styles/theme/defaultTheme.ts

```diff
@@ -461,9 +461,14 @@ export const defaultTheme = createTheme({
       styleOverrides: {
         root: ({ theme }) => ({
           borderRadius: '24px',
-          '.MuiInputBase-root > :first-child': {
+          '.MuiInputBase-root': {
             // Prevent first child's edges overflowing due to border-radius
-            marginLeft: theme.spacing(4),
+            '.MuiInputAdornment-positionStart': {
+              marginLeft: theme.spacing(4),
+            },
+            '.MuiInputBase-input:not(.MuiInputBase-inputAdornedStart)': {
+              marginLeft: theme.spacing(4),
+            },
```

Comment:
I don't get this. This the new code, aren't we applying the same spacing to the start position and all the non-start positions?

Reply:
Refactored to make it clearer that it applies spacing to the first child:
```ts
            // Prevent first child's edges overflowing due to border-radius
            '&.MuiInputBase-adornedStart .MuiInputAdornment-positionStart': {
              marginLeft: theme.spacing(4),
            },
            '&:not(.MuiInputBase-adornedStart) .MuiInputBase-input': {
              marginLeft: theme.spacing(4),
            },
```